app/data/organizations.js

```diff
@@ -11,7 +11,7 @@ export default {
       },
       {
         type: 'gitlab',
-        identifier: 'coala',
+        identifier: 'coala/mobans',
```

Comment:
This is obviously wrong, as it is not an org identifier.  It is a project identifier.

This is not too different from the problem I raised in the PR a month ago.
https://github.com/coala/git-task-list/pull/44#discussion_r194411943

`fetchGitlabProjects` still doesnt get the projects for the org.

Reply:
Identifier in gitlab currently only support per repository, however, in github, only support per organization.

In gitlab, supporting per org, will make multiple request. 

It fit nicely with wikidata where, it already support github organization, but not gitlab. It can be identified via issue tracker used, which is currently support full link.